Gemfile

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
+gem 'byebug'
```

Comment:
This should be added to the `:development` and `:test` groups only so that we don't install this in production. 🙀